docs/source/daml/code-snippets/Structure.daml

```diff
@@ -30,20 +30,32 @@ template NameOfTemplate
     -- end of template outline snippet
     -- start of choice snippet
     -- option 1 for specifying choices: choice name first
-    choice NameOfChoice :
-          () -- replace () with the actual return type
-        with
+    choice NameOfChoice
+      : () -- replace () with the actual return type
+      with
         party : Party -- parameters here
       controller party
-        do
-          return () -- replace this line with the choice body
+      do
+        return () -- replace this line with the choice body
 
     -- option 2 for specifying choices (deprecated syntax): controller first
     controller exampleParty can
-      NameOfAnotherChoice :
-          () -- replace () with the actual return type
+      NameOfAnotherChoice
+        : () -- replace () with the actual return type
         with
           party : Party -- parameters here
         do
           return () -- replace the line with the choice body
       -- end of choice snippet
+
+    -- start of choice observer snippet
```

Comment:
This was previously defined in `docs/source/daml/code-snippets-dev/Structure.daml`, but since the choice observers feature has long been released, it didn't make sense to keep both files.